lua/weapons/kilos_combat_toolgun.lua

```diff
@@ -0,0 +1,127 @@
+SWEP.PrintName = "Kilo's Combat Tool Gun" -- The name of the weapon
+SWEP.Author = "Kilo"
+SWEP.Purpose = "For when circumstances exceed that of the regular tool gun."
+SWEP.Category = TASWeapons.Category
+
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+
+SWEP.Base = "weapon_base"
+
+local ShootSound = Sound("weapons/airboat/airboat_gun_lastshot2.wav")
```

Comment:
I think this'd be better as a table entry into the SWEP, and the sound path turned into a constant.